### ISSUE TITLE
Bug in met-7.0 grid_stat.  Segfault when running make test on WCOSS Hank.

### ISSUE THREAD

**Issue by dwfncar** [dtcenter/MET#978]
Julie ran into a segfault on Hank when running make test for met-7.0 on hank.  We found/fixed a bug in grid_stat.cc starting on line 601.
<br/>

<br/>
This bug was introduced in met-7.0 when I switched from using one top-level output_flag setting for all vx tasks to specifying it separately for each vx task.
<br/>

<br/>
When allocating memory for neighborhood methods, I checked "conf_info.vx_opt[i].output_flag" when I should have checked "conf_info.output_flag" which contains a summary of the settings across all vx tasks.
<br/>

<br/>
Basically, that "i" is an uninitialized value at that point in the logic.  On WCOSS Cray, it was initialized to a very large value while all the other compilers likely initialized it to 0.
<br/>

<br/>
This underscores the need to test each release using the Intel compilers with the flag that initializes variables to very weird values.
<i>[MET-978] created by johnhg</i>
